specs/darwin/install_history.table

```diff
@@ -0,0 +1,11 @@
+table_name("install_history")
+description("OS X package install history.")
+schema([
+    Column("date", INTEGER, "Label date as UNIX timestamp"),
+    Column("display_name", TEXT, "Label displayName"),
+    Column("display_version", TEXT, "Label displayVersion"),
+    Column("process_name", TEXT, "Label processName"),
```

Comment:
On my host this is only `installer` or `storedownloadd`, maybe this is more indicative of the package install source or reason-- would `install_source` or `source` be more appropriate?